roles/openshift_repos/tasks/main.yaml

```diff
@@ -1,13 +1,20 @@
 ---
+- set_fact:
+    deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+    openshift_deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
```

Comment:
@detiber wanted to get your opinion on whether this is the right place to ensure these both default to each other. `openshift_facts` and `openshift_repos` do not depend on each other at this time. Maybe do it in both places?